helm/hpcc/values.schema.json

```diff
@@ -237,6 +237,14 @@
         "noResourceValidation": {
           "type": "boolean"
         },
+        "stubInstanceResources": {
+          "$ref": "#/definitions/stubInstanceResources"
+        },
+        "stubPerInstanceCPU": {
```

Comment:
Is this used? If so how does it differ from stubInstanceResources.cpu ?

Reply:
no, was left in by mistake, will remove.